src/components/WorkflowForms/NewClientSignup/GatherInfo/hooks/useProviders.js

```
@@ -0,0 +1,269 @@
+
+  function loadProviderTypes() {
+    return api.get_healthcare_provider_types()
+      .then((result = {}) => {
+         if (result && result.success === true) {
+            const medicalTypes = result.data.map(item => ({
+              value: item.xref || item.id,
+              label: item.name || item.description || item.xref
+            }));
+            setProviderTypes(medicalTypes);
+         } else {
+         }
```

ESLint

[error] Empty block statement.

(no-empty)